department/department_app/models/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Models"""
```

Comment:
Please use only absolute imports in your project. Relative imports can cause many unexpected problems in the future :)

Reply:
fixed